docs: escape comma (#655)
jina-ai/finetuner: docs/walkthrough/create-training-data.md

Please note, that local files can not be processed by the Finetuner if you deactivate `convert_to_blob` or `create_point_clouds`.
```

```{important} 

> If the text field use commas, it breaks the CSV format since it is interpreted as a new column.

Reviewer: ```suggestion
If the text field contains commas, it breaks the CSV format since it is interpreted as spanning over multiple columns.
```

> In this case, please enclose the field in double quates, such as `field1,"field, 2",field3, ...`.

Reviewer: ```suggestion
In this case, please enclose the field in double quotes, such as `field1,"field, 2",field3, ...`.
```

> ```{important} 
> If the text field use commas, it breaks the CSV format since it is interpreted as a new column.
> In this case, please enclose the field in double quates, such as `field1,"field, 2",field3, ...`.
> ```

Reviewer 2: ```suggestion
```{important} 
If the text field use commas, it breaks the CSV format since it is interpreted as a new column.
In this case, please enclose the field in double quotes, such as`"field, with a comma", field without a comma`.
```
Since the user should only have two fields per row it might be confusing to have 3+ fields per row in the example
```
